components/scream/src/physics/shoc/shoc_ic_cases.cpp

```diff
@@ -202,8 +202,8 @@ FortranData::Ptr make_standard(const Int shcol, Int nlev, Int num_qtracers) {
     // Compute pressure differences and host_dse / exner.
     for (Int k = 0; k < nlev; ++k) {
       d.pdel(i, k) = std::abs(d.presi(i, k+1) - d.presi(i, k));
-      d.exner(i, k) = pow(d.pres(i, k)/consts::P0, consts::Rair/consts::Cpair);
-      d.host_dse(i, k) = consts::Cpair * d.exner(i, k) * d.thv(i, k) +
+      d.inv_exner(i, k) = 1.0/pow(d.pres(i, k)/consts::P0, consts::Rair/consts::Cpair);
+      d.host_dse(i, k) = consts::Cpair * d.thv(i, k)/d.inv_exner(i, k) +
```

Comment:
This change causes the `shoc_run_and_cmp` tests to fail. The baselines are created with the old version where the `inv_exner` passed to `shoc_main()` (F90) actually was `exner`. Since C++ and F90 were doing the same computation, they matched, even though the value for `inv_exner` (in f90) and the computation with `exner` (c++) didn't make sense.

Reply:
Is the assumption being made here that `phis` is always 0.0?  See this line: https://github.com/E3SM-Project/scream/blob/ef54333db4acd190dabcadb299fb506f2f98b56e/components/eam/src/physics/cam/shoc.F90#L3798

Reply:
I guess so, looking at the line just below.